app/services/calculator.rb

```diff
@@ -0,0 +1,45 @@
+class Calculator
+  def initialize(entries)
+    @time_entries = entries
+  end
```

Comment:
Below you use parameters called `time_entries`. Probably it shouldn't bug me, but it does somehow...